CExcercises/Ex1-nIndirections.c

```diff
@@ -16,22 +16,23 @@ El resultado será 3.*/
 int nIndirections(void* initial, unsigned int indirections);
 
 int main (int argc, char **argv)
-{
+{	
+	int indirections = 2; // To not use magical values
```

Comment:
be careful with the types that you choose, in this particular
case use unsigned int